packages/commands/src/route-tokens-to-user-bank.mjs

```diff
@@ -0,0 +1,249 @@
+import {
+  TOKEN_PROGRAM_ID,
+  createAssociatedTokenAccountInstruction,
+  createTransferCheckedInstruction,
+  getAccount,
+  mintTo
+} from '@solana/spl-token'
+import {
+  Connection,
+  Keypair,
+  PublicKey,
+  Transaction,
+  TransactionInstruction,
+  sendAndConfirmTransaction
+} from '@solana/web3.js'
+import chalk from 'chalk'
+import { Option, program } from 'commander'
+
+import { route } from '@audius/spl'
+
+import { initializeAudiusLibs } from './utils.mjs'
+
+const TOKEN_DECIMALS = {
+  audio: 8,
+  usdc: 6
+}
+
+const MEMO_PROGRAM_ID = new PublicKey(
+  'Memo1UhkJRfHyvLMcVucJwxXeuD728EqVDDwQDxFMNo'
+)
+
+program
+  .command('route-tokens-to-user-bank')
+  .argument('<amount>', 'The amount of tokens to send')
+  .description(
+    'Transfer $USDC or $AUDIO tokens using payment router into a user bank from the owner wallet'
+  )
+  .addOption(
+    new Option('-m, --mint [mint]', 'The currency to use')
+      .choices(['audio', 'usdc'])
+      .default('usdc')
+  )
+  .option('-f, --from <from>', 'The account to send tokens to (handle)')
+  .option('--memo <memo>', 'A data string to attach as a memo')
+  .action(async (amountInput, { from, mint, memo }) => {
+    const amount = BigInt(amountInput)
+    const audiusLibs = await initializeAudiusLibs(from)
+    const { solanaWeb3Manager } = audiusLibs
+
+    const { userbank: userbankPublicKey } =
+      await solanaWeb3Manager.createUserBankIfNeeded({
+        mint
+      })
+
+    if (!process.env.SOLANA_ENDPOINT) {
+      program.error('SOLANA_ENDPOINT environment variable not set')
+    }
+
+    const connection = new Connection(process.env.SOLANA_ENDPOINT)
+    const feePayer = Keypair.fromSecretKey(
+      Uint8Array.from(JSON.parse(process.env.SOLANA_FEEPAYER_SECRET_KEY))
+    )
+
+    const paymentRouterPublicKey = new PublicKey(
+      process.env.SOLANA_PAYMENT_ROUTER_PUBLIC_KEY
+    )
+
+    const tokenMint =
+      mint === 'audio'
+        ? new PublicKey(process.env.SOLANA_TOKEN_MINT_PUBLIC_KEY)
```

Comment:
not your code but this should probably be renamed to `SOLANA_AUDIO_TOKEN_MINT_PUBLIC_KEY` eventually

Reply:
Agreed. Many of the env vars don't follow the same naming convention. Changing this feels dangerous though as it's consumed in a lot of packages 😅